tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1432,32 +1431,16 @@ def test_generate_compilation_all_outputs(self):
 @require_torch
 @require_torchaudio
 class WhisperModelIntegrationTests(unittest.TestCase):
-    def setUp(self):
-        self._unpatched_generation_mixin_generate = transformers.GenerationMixin.generate
```

Comment:
this was only used in one test, and that test should use the base `generate`

idk why it was [added in the first place](https://github.com/huggingface/transformers/pull/29225) 🤔

Reply:
#29225 but i have no idea with it.

Reply:
It is linked to #29312, but the change here is only test. So good for me if it works.

Reply:
That is implicitly tested: the output checked in the test is different if `num_beams` is not respected :)

(just like in all other beam search integration tests: we check the output, which is sensible enough to detect bad usage of flags)